built_value_generator/lib/src/value_source_class.dart

```diff
@@ -203,6 +230,16 @@ abstract class ValueSourceClass
       result.add('Make class abstract.');
     }
 
+    if (hasBuiltValueImportWithShow) {
+      result.add('Stop using "show" when importing '
+          '"package:built_value/built_value.dart".');
```

Comment:
Maybe add a reason for it?

Reply:
Done: 'it prevents the generated code from finding helper methods'